.github/workflows/inspektor-gadget.yml

```diff
@@ -52,6 +52,7 @@ jobs:
     - name: Check if generated files are updated
       run: |
         make manifests generate generate-documentation
+        make -C tools/syscalls-update
```

Comment:
The generated file doesn't depend on a source code stored on this repo but on an external file. I think it could be confusing to have this job failing on a PR that doesn't have any relationship with this. Is it difficult to change the approach to have a bot that opens a PR to update this file?

Reply:
Ok, I'll remove it. The cron job can be in another PR.